test/src/module/common/ioHttpTest.c

```diff
@@ -406,28 +427,59 @@ testRun(void)
                 hrnTlsServerAccept();
 
                 hrnTlsServerExpectZ("GET / HTTP/1.1\r\n\r\n");
-                hrnTlsServerReplyZ("HTTP/1.1 404 Not Found\r\ncontent-length:0\r\n\r\n");
```

Comment:
removing content-length:0 is changing the test

Reply:
Added `content-length:0` to the test at line 324.